tests/libtest.sh

```diff
@@ -363,8 +363,15 @@ setup_os_boot_grub2() {
 
 setup_os_repository () {
     mode=$1
-    bootmode=$2
     shift
+    bootmode=$1
+    shift
+    if test -n "${1:-}"; then
+        bootdir=${1}
+        shift
+    else
+        bootdir=usr/lib/ostree-boot
+    fi
```

Comment:
I think I didn't do that since we'd lose the `shift`, but eh, we can deal with that if we start adding more arguments.